### PR TITLE
1556 broken agent manual run Windows

### LINKED ISSUE #1556: Broken agent manual run as windows user
## Describe the bug
If you select windows and input a username, the generated command will not work:
image

image

## To Reproduce
1. Configure the run command as shown in the screenshot
2. See error

## Solutions:
These changes need to be done to the command:
1. Absolute path needs to be provided instead of "./monkey" as the output of downloaded file
2. `powershell.exe` needs to be under `-FilePath` flag: `-FilePath powershell.exe`
3. The `$true` needs to be escaped with ` character.

Example of a working command: 
```
Start-Process -FilePath powershell.exe -ArgumentList "-noexit [System.Net.ServicePointManager]::ServerCertificateValidationCallback = {`$true}; (New-Object System.Net.WebClient).DownloadFile('https://172.19.176.1:5000/api/monkey/download/monkey-windows-64.exe','C:\Users\user3\monkey.exe'); Start-Process -FilePath 'C:\Users\user3\monkey.exe'
 -ArgumentList 'm0nk3y -s 172.19.176.1:5000';" -Credential user3
```

## Tasks
- [ ] Fix it. (0.5d) - @VakarisZ

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do? 

Fixes #1556 
Removes "manual run from CMD" option
Fixes monkey to be downloaded to temp rather than working directory

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing? 
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Have you successfully tested your changes locally? Elaborate:
    > Tested by running powershell command
    > Tested by running powershell command as a user
    > Tested by running wget and curl commands
image

### CODE REVIEW

**Review by ilija-lazoroski — APPROVED** [review]
Looks good to me. Nice job!

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_windows_powershell.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
 import {OS_TYPES} from '../utils/OsTypes';
 
 
-export default function generateLocalWindowsPowershell(ip, osType, username) {
+export function getAgentDownloadCommand(ip, osType) {
```
Comment (shreyamalviya) [inline monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_windows_powershell.js:4]: If we're renaming this function here, let's do it in the other two places too.

**Review by shreyamalviya — APPROVED** [review]

**Comment by mssalvatore** [comment]
Missing changelog entries (requires 2).

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_windows_powershell.js (hunk 1 above)
Comment (VakarisZ) [reply monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_windows_powershell.js]: This function is still here, I simply extracted the command formatting into a separate function

**Review by mssalvatore — CHANGES_REQUESTED** [review]
We need to revert the Linux changes as they introduce a security vulnerability and test that the preexisting linux command works properly with `su`.

**Review by mssalvatore — APPROVED** [review]

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_windows_powershell.js (hunk 1 above)
Comment (shreyamalviya) [reply monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_windows_powershell.js]: Oh. 🤦‍♀️ My bad.

**Pull request merged**